src/views/single-event/recurring-description.php

```diff
@@ -22,7 +22,7 @@
 
 	<span><?php echo $recurrence_description ?></span>
 
-	<a href="<?php echo esc_url( tribe_all_occurences_link( $post_id, false ) ) ?>">
+	<a href="<?php echo esc_url( tribe_all_occurrences_link( $post_id, false ) ) ?>">
```

Comment:
Not sure if we can fix this one, since it's a method that would need to confirm that typo doesn't exist on the method.

Reply:
Reverted 🍏